source/aggregation.txt

```diff
@@ -0,0 +1,205 @@
+      :start-after: start-match-group
+      :end-before: end-match-group
+      :language: php
+      :dedent:
+
+   .. output:: 
+      :visible: false
+
+      {"_id":"Brooklyn","count":173}
+      {"_id":"Queens","count":204}
+      {"_id":"Bronx","count":71}
+      {"_id":"Staten Island","count":20}
+      {"_id":"Missing","count":2}
+      {"_id":"Manhattan","count":221}
+
+Explain an Aggregation
+~~~~~~~~~~~~~~~~~~~~~~
+
+To view information about how MongoDB executes your operation, you can
+instruct the MongoDB query planner to **explain** it. When MongoDB explains
+an operation, it returns **execution plans** and performance statistics.
+An execution plan is a potential way in which MongoDB can complete an operation.
+When you instruct MongoDB to explain an operation, it returns both the
+plan MongoDB executed and any rejected execution plans.
+
+To explain an aggregation operation, run the ``explain`` database command by passing
+the command information to the ``MongoDB\Database::command()`` method. You must specify the
+``aggregate``, ``pipeline``, and ``cursor`` fields in the ``explain`` command document
+to explain the aggregation.
+
+The following example instructs MongoDB to explain the aggregation operation from the
+preceding :ref:`php-aggregation-example`:
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/aggregation.php
+      :start-after: start-explain
+      :end-before: end-explain
+      :language: php
+      :dedent:
+
+   .. output:: 
+      :visible: false
+
+      {"explainVersion":"2","queryPlanner":{"namespace":"sample_restaurants.restaurants",
+      "indexFilterSet":false,"parsedQuery":{"cuisine":{"$eq":"Bakery"}},"queryHash":"865F14C3",
+      "planCacheKey":"D56D6F10","optimizedPipeline":true,"maxIndexedOrSolutionsReached":false,
+      "maxIndexedAndSolutionsReached":false,"maxScansToExplodeReached":false,"winningPlan":{
+      ... }
+
+
+Additional Information
+----------------------
+
+To view a tutorial that uses the {+php-library+} to create complex aggregation
+pipelines, see `Complex Aggregation Pipelines with Vanilla PHP and MongoDB 
+<https://www.mongodb.com/developer/products/mongodb/aggregations-php-mongodb/>`__
+in the MongoDB Developer Center.
+
+MongoDB Server Manual
+~~~~~~~~~~~~~~~~~~~~~
+
+To learn more about the topics discussed in this guide, see the following
+pages in the {+mdb-server+} manual:
+
+- To view a full list of expression operators, see :manual:`Aggregation
+  Operators </reference/operator/aggregation/>`.
+
+- To learn about assembling an aggregation pipeline and to view examples, see
+  :manual:`Aggregation Pipeline </core/aggregation-pipeline/>`.
+
+- To learn more about creating pipeline stages, see :manual:`Aggregation
+  Stages </reference/operator/aggregation-pipeline/>`.
+
+- To learn more about explaining MongoDB operations, see
+  :manual:`Explain Output </reference/explain-results/>` and
+  :manual:`Query Plans </core/query-plans/>`.
+
+.. TODO:
```

Comment:
Does the `php-aggregation-tutorials-landing` target exist already? I'm not sure how `.. TODO:` is interpreted here, and whether that prevents the subsequent content from rendering.

Reply:
Not yet, so this section is commented out (and will be uncommented when the aggregation tutorials are created)